beard/metrics/tests/test_clustering.py

```diff
@@ -24,27 +24,33 @@
 def test_precision_recall_fscore():
     """Test the results of precission_recall_fscore."""
     # test for the border case where score maximum
-    y = np.array([1, 2, 1, 3, 2, 4, 5, 4])
-    assert clustering.paired_precision_recall_fscore(y, y) == (1, 1, 1)
+    y = [1, 2, 1, 3, 2, 4, 5, 4]
+    assert_equal(clustering.paired_precision_recall_fscore(y, y), (1, 1, 1))
```

Comment:
Because of `check_clusterings`, both lists and arrays are accepted as inputs. I suggested to switch to lists in tests, because @etzemis  was mixing both previsouly, which was not very elegant...
See eg https://github.com/inveniosoftware/beard/blob/master/beard/metrics/tests/test_clustering.py#L69